main/src/ca/uwaterloo/flix/language/fmt/FormatType.scala

```diff
@@ -263,7 +289,7 @@ object FormatType {
       case SimpleType.PureArrow(arg, ret) =>
         val argString = delimitFunctionArg(arg)
         val retString = delimit(ret, Mode.Type)
-        s"$argString -> $retString"
+        s"$argString -> $retString \\ {}"
```

Comment:
The "right" way to do this is to change the transformation to use `PolyEffArrow` with `Empty` in the eff.

Reply:
Are you sure? The function is pure afterall?

Reply:
I think so. The idea of `SimpleType` is that it closely resembles the format of the source code type.

Reply:
Here, `a -> b \ {}` could correspond to `SimpleType.PureArrow` or `SimpleType.PolyArrowWithEff`.

It's not a huge problem since we don't parse in that way, but less principled.